src/test/java/seedu/duke/template/LcTemplateTest.java

```diff
@@ -2,14 +2,109 @@
 
 import org.junit.jupiter.api.Test;
 import seedu.duke.DukeException;
+import seedu.duke.component.Capacitor;
 
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 
 class LcTemplateTest {
+    public static final double DELTA = 1e-15;
 
     @Test
     void calcImpedance_zeroCapacitance_expectException() {
         LcTemplate t = new LcTemplate();
         assertThrows(DukeException.class, t::calcImpedance);
     }
+
+    @Test
+    void calcImpedance_returnsDouble() throws DukeException {
```

Comment:
Consider changing all method names to include the test input/scenario.
https://se-education.org/guides/conventions/java/basic.html#naming

Reply:
Got it, will rename accordingly!